resources/services/cloudrun/services_mock_test.go

```diff
@@ -0,0 +1,65 @@
+package cloudrun
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/cloudquery/cq-provider-gcp/client"
+	"github.com/cloudquery/faker/v3"
+	"github.com/julienschmidt/httprouter"
+	"google.golang.org/api/option"
+	"google.golang.org/api/run/v1"
+)
+
+func createServicesServer() (*client.Services, error) {
+	ctx := context.Background()
+	services := make([]*run.Service, 1)
+	if err := faker.FakeData(&services[0]); err != nil {
+		return nil, err
+	}
+	gen := faker.FakeDataWithNilPointerGenerator()
```

Comment:
This is the first time we're using the new `FakeDataWithNilPointerGenerator` feature added in https://github.com/cloudquery/faker/pull/5, but it would be good to use it in other places too to catch nil-pointer dereference bugs.